src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -745,6 +747,73 @@ class StringGenerator : public Generator<std::string> {
     const size_t _alphabetLength;
 };
 
+/** `{^ChooseFromDataset:{...}` */
+class RandomDataFromDataset : public Generator<std::string> {
```

Comment:
Should this have `string` somewhere in its name to clarify that it works for string datasets, not numeric ones?

Reply:
Renamed to RandomStringFromDataset.